metrics/mixin/alerts/perf.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ceph-perf-warnings.rules',
```

Comment:
There's already a new group for perf alerts added by #2309 . Please re-use that.

Reply:
it has a group `mds-performance-alerts.rules`, should we rename that
or introduce a new group for osd?

Reply:
Make it `ceph-daemon-performance-alerts.rules` maybe.